Hardware/Hardware/Printer/AirPrintReceipt/ReceiptRenderer.swift

```diff
@@ -250,3 +251,19 @@ private extension ReceiptRenderer {
         )
     }
 }
+
+private extension String {
+    func stripHtml() -> String {
```

Comment:
This function returns a modified copy of the original string, without modifying the original. In those cases, the convention in Swift is to name the method/property ending in -ed (participle tense?).

To me, `htmlStripped()` would sound "Swiftyer"

Reply:
Makes a lot of sense. Not sure why we don't call them like that in kotlin

Fixed: 7c2f860b17eddc315b5b25356605532c2b18e119